src/Transfer/Resources/Database/Attributes/BoolAttribute.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class BoolAttribute extends Attribute
```

Comment:
```suggestion
class Bool extends Attribute
```

No need of the Attribute suffix. Make sure to change it in the file name.